docker/fedora-41-nightly.Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN dnf install -y \
   clang-tools-extra
 
 # Bust the cache
-ARG STAMP=1729535688
+ARG STAMP=1731945952
```

Comment:
Honestly surprised this is needed for GH actions, I always thought this was a workaround for broken image caching in Cirrus CI.

Reply:
I update those once in a while locally if I want to bust the cache. I'll revert.